### PR TITLE
🗞️ Updates to auto-completion to reflect current engine context

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: src/components/form/textinput/TextInputElement.tsx [hunk 1]
```diff
@@ -110,16 +110,16 @@ export class TextInputElement extends React.Component<TextInputProps, TextInputS
...
-      'date',
+      'child',
       'fields',
+      'input',
+      'parent',
+      'results',
+      'run',
+      'trigger',
       'urns',
-      'input'
```
Comment (rowanseymour) [inline src/components/form/textinput/TextInputElement.tsx]: adding webhook and sorting a-z so it's easier to compare with goflow's list

**Comment by rowanseymour** [comment]
@ericnewcomer turns out I don't quite understand how the snapshot tests work and how to update them etc...

**Comment by ericnewcomer** [comment]
I'll take this and also spend some time cleaning up the README

**Comment by rowanseymour** [comment]
@ericnewcomer won't be insulted if it's easier to start from scratch. @nicpottier and I chatted a bit about this at lunch and the end goal should be goflow generating a JSON map of the context from comments on code, and the editor consuming that.. but that's probably for another day.

**Comment by ericnewcomer** [comment]
Totally agree on this being json!

**Pull request merged**